src/lib/transaction_logic/mina_transaction_logic.ml

```diff
@@ -546,7 +546,17 @@ module type S = sig
     -> ledger
     -> bool Or_error.t
 
-  module For_tests : sig
+    module For_tests : sig
+      module Stack (Elt : sig type t end) : sig
+        type t = Elt.t list
+        val if_ : bool -> then_:'a -> else_:'a -> 'a
```

Comment:
It probably is `if_ : bool -> then_:t -> else_:t -> t` ?

Reply:
Fixed with [9103b44](https://github.com/MinaProtocol/mina/pull/12796/commits/9103b44dffae591f1190cc9d457dffc25b7fab88).